skyrim-platform/src/platform_se/skyrim_platform/GameEventSinks.cpp

```diff
@@ -84,6 +85,36 @@ RE::BSEventNotifyControl GameEventSinks::ProcessEvent(
   return RE::BSEventNotifyControl::kContinue;
 }
 
+RE::BSEventNotifyControl GameEventSinks::ProcessEvent(
+  const RE::TESWaitStartEvent* event,
+  RE::BSTEventSource<RE::TESWaitStartEvent>* eventSource)
+{
+  auto converted =
+    reinterpret_cast<const TESEvents::TESWaitStartEvent*>(event);
+
+  if (!converted) {
+    return RE::BSEventNotifyControl::kContinue;
+  }
+
+  auto waitStart = converted->waitStartTime;
+  auto waitEnd = converted->desiredWaitEndTime;
+
+  SkyrimPlatform::GetSingleton().AddUpdateTask([waitStart, waitEnd] {
+    auto obj = JsValue::Object();
+
+    if (waitStart < 0 || waitEnd < 0) {
```

Comment:
What's the meaning of this condition? Seems to be always false.
Please add a comment if this check is needed or remove.